File: fizz.rb
#!/usr/bin/env ruby
#
#######################################################################
#                              FuzzyBees                              #
#######################################################################

1.upto(20) do |fuzzybee|
	if (fuzzybee % 15).zero?
		puts "FuzzyBees!"
	elsif (fuzzybee % 3).zero?
		puts "Fizz"
	elsif (fuzzybee % 5).zero?
		puts"Buzz"
	else
		puts fuzzybee
	end
end

File: fizz.py
#!/usr/bin/env python

#######################################################################
#                              FuzzyBees                              #
#######################################################################

for fuzzybee in xrange(1,21):
    if fuzzybee % 15 == 0:
        print "FuzzyBees!"
    elif fuzzybee % 3 == 0:
        print "Fizz"
    elif fuzzybee % 5 == 0:
        print "Buzz"
    else:
        print fuzzybee
